great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -1,3 +1,6 @@
+import inspect
+import os.path
```

Comment:
Nitpick - could we just import `os` to be consistent with the rest of the codebase?